src/components/block-button.js

```diff
@@ -0,0 +1,31 @@
+window.blockUser = function(id) {
+  NAF.connection.adapter.publisher.handle.sendMessage({ kind: "block", whom: id });
+  NAF.connection.entities.removeEntitiesOfClient(id);
```

Comment:
The adapter's block funciton above should probably just emit an even so that it gets handled by the handler you already have doing this.

Reply:
👍